distributed/tests/test_scheduler.py

```diff
@@ -4159,6 +4159,23 @@ def __init__(self, instance=None):
     assert s.plugins["nonidempotentplugin"].instance == "second"
 
 
+@gen_cluster(nthreads=[])
```

Comment:
Tests are currently spread around a bit, but the majority of tests for the `SchedulerPlugin` can be found in `distributed/diagnostics/tests/test_scheduler_plugin.py`. I'd suggest moving this test over for better discovery.